src/OnboardingSPA/components/LivePreviewSkeleton/index.js

```diff
@@ -0,0 +1,38 @@
+import { useSelect } from '@wordpress/data';
+import { LivePreviewSelectableCard } from '../LivePreview';
+
+/**
+ * Renders Skeletons for Live Previews.
+ *
+ * @property {number} count                     The number of Live Previews to be shown
+ * @property {string} className                 The class name for the Live Preview
+ * @property {number} viewportWidth             Viewport Width for the Live Preview
+ * @property {number} skeletonLoadingTime       Change the Animation time for the Skeleton
+ */
+const LivePreviewSkeleton = ({ count, className, viewportWidth, skeletonLoadingTime }) => {
```

Comment:
Like we discussed can this accept a callback function and only remove the skeleton once that is complete?